### ISSUE TITLE
A Work has a longitude

### ISSUE THREAD

**Issue by no-reply** [UCLALibrary/californica#104]
Add a `longitude` attribute for the `Work` class:

  - [ ] A Work object can have a `longitude` field
  - [ ] The predicate for it is as determined in #103
  - [ ] The label for it is "Longitude" (?)

Depends on #103.

**Comment by mark-dce** [comment]
superseded by #150
